charts/spire/charts/spire-server/templates/tornjak-ingress.yaml

```
@@ -0,0 +1,16 @@
+{{- if .Values.tornjak.ingress.enabled -}}
+{{- $fullName := include "spire-tornjak.fullname" . -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  namespace: {{ include "spire-server.namespace" . }}
+  labels:
+    {{ include "spire-server.labels" . | nindent 4}}
+  {{- with .Values.tornjak.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{ include "spire-lib.ingress-spec" (dict "ingress" .Values.tornjak.ingress "svcName" (include "spire-tornjak.servicename" .) "port" .Values.tornjak.ingress.servicePort "global" .Values.global) | nindent 2 }}
```

```suggestion
  {{ include "spire-lib.ingress-spec" (dict "ingress" .Values.tornjak.ingress "svcName" (include "spire-tornjak.servicename" .) "port" "tornjak-srv-http" .Values.global) | nindent 2 }}
```

This uses the service port by name as defined in the Service definition.

---

"global" got accidentally dropped in the suggestion?

---

Forgot to remove the other global part, as there isn't a value required with this suggestion. Thanks for noticing my mistake.